src/quo/previews/preview.cljs

```diff
@@ -39,6 +39,13 @@
 (def label-style {:flex          0.4
                   :padding-right 8})
 
+(defn label-view [state label]
+  [rn/view {:style label-style}
+   [quo/text
+    (when-let [label-color (:preview-label-color @state)]
+      {:style {:color label-color}})
+    label]])
+
```

Comment:
Some components like the bottom tab and switcher cards only support dark mode. And in the light mode text color is also black.
So this allows to override label-color to make it white on the hard-coded black background.

Reply:
could you elaborate please on why this change in quo ?

Reply:
hi @flexsurfer, thank you for reviewing the PR.

Even for quo2 preview screens, we are using `quo/previews/preview.cljs`. This change will not affect quo screens as we are only changing label-color value when we have passed it.

But still, maybe we can duplicate this file in quo2 and use it from there, So any changes don't affect quo screens. what do you recommend?

Reply:
i would say, im fine with any solution, so its up to you